src/views/DocList.tsx

```diff
@@ -3,9 +3,21 @@
 export default function DocList(){
     return (
         <div>
+<<<<<<< HEAD
```

Comment:
Just like the other files, it looks like there are git conflicts on this page. Please resolve all conflicts